kernel/libs/aster_page_allocator/src/buddy_allocator.rs

```diff
@@ -0,0 +1,236 @@
+// SPDX-License-Identifier: MPL-2.0
```

Comment:
As @tatetian points out in the comments of #1317 , do not include license or source origin information as Rust doc. Just make them regular comments. Second, it does not say the original code is licensed under MIT. Third, it is unclear which parts are changed by us, which is protected by our copyright license (MPL).

So change the file preamble to the following form.

```rust
// The original source code is from [buddy_system_allocator](https://github.com/rcore-os/buddy_system_allocator),
// which licensed under the following license.
//
// SPDX-License-Identifier: MIT
//
// Copyright (c) 2019-2020 Jiajie Chen
//
// We make the following new changes:
// * Change 1
// * Change 2
//
// These changes are released under the following license:
//
// SPDX-License-Identifier: MPL-2.0
```